src/components/AttachmentPicker/index.native.js

```diff
@@ -62,12 +63,21 @@ const documentPickerOptions = {
   * @return {Object}
```

Comment:
Oh also this needs to be updated please :)

```suggestion
  * @return {Promise}
```